components/contentblocks/ActionStatusGraphsBlock.tsx

```diff
@@ -39,12 +40,8 @@ interface Props
   categoryId?: string;
 }
 
-const ActionStatusGraphsBlock = ({
-  id = '',
-  categoryId,
-  columnProps,
-  ...graphsProps
-}: Props) => {
+const ActionStatusGraphsBlock = (props: Props) => {
+  const { id = '', categoryId, columnProps, ...graphsProps } = props;
```

Comment:
I prefer doing this as it often helps debugging (logging all props)

Reply:
You could just log `graphsProps` or add `...rest` in cases where we don't already destructure the remaining props. I find this makes components harder to refactor because it's harder to spot if a prop is used or not